aether-kernel/aether/kernel/api/models.py

```diff
@@ -473,7 +280,19 @@ class Entity(ExportModelOperationsMixin('kernel_entity'), models.Model):
     projectschema = models.ForeignKey(to=ProjectSchema, on_delete=models.SET_NULL, null=True)
     submission = models.ForeignKey(to=Submission, on_delete=models.SET_NULL, blank=True, null=True)
 
+    # redundant but speed up queries
+    project = models.ForeignKey(to=Project, on_delete=models.SET_NULL, blank=True, null=True)
```

Comment:
Should it be possible to create an entity which does not belong to a Project? When would that be useful? Asking since it's usually helpful to have as few nullable fields as possible.

Reply:
It might not but because `projectschema` and `submission` were already optional... :woman_shrugging:

Reply:
This is something we need to discuss, the `on_delete=models.SET_NULL` option indicates that if the foreign instance it's deleted the entity will remain in the database.

Reply:
Right, it makes sense that `project` is optional if `projectschema` and `submission` are. As I've understood the system, only `submission` need to be nullable (an entity might be submitted directly, bypassing entity extraction) but `projectschema` is always required (an entity is expected to conform to its `self.projectschema.schema`).

Reply:
In that case, might it be better to link the entity directly to the schema? or maybe do not allow to delete a project schema, or a project, or a schema if they have at least one linked entity? `on_delete=models.PROTECT` :thinking: 

@shawnsarwar @quesebifurcan

Reply:
Maybe keep it this way for now and address this issue in a separate PR?